blocks/library/latest-posts/index.php

```diff
@@ -62,28 +65,48 @@ function gutenberg_render_block_core_latest_posts( $attributes ) {
 	return $block_content;
 }
 
-register_block_type( 'core/latest-posts', array(
-	'attributes'      => array(
-		'postsToShow'     => array(
-			'type'    => 'number',
-			'default' => 5,
-		),
-		'displayPostDate' => array(
-			'type'    => 'boolean',
-			'default' => false,
-		),
-		'layout'          => array(
-			'type'    => 'string',
-			'default' => 'list',
-		),
-		'columns'         => array(
-			'type'    => 'number',
-			'default' => 3,
-		),
-		'align'           => array(
-			'type'    => 'string',
-			'default' => 'center',
-		),
-	),
+register_block_type('core/latest-posts', array(
 	'render_callback' => 'gutenberg_render_block_core_latest_posts',
-) );
+	'attributes'      =>
+		array(
```

Comment:
Why was this changed to put array opening on a line of its own? This isn't a style I've seen common to WordPress, and adds quite a bit of excessive indentation and awkward arrow spacing.

Reply:
It was a try to make things look more aligned while respecting phpcs rules. I reverted this change but unfortunately, I'm not seeing a way to make things more aligned and respect phpcs.